cypress/support/component-index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
what this?

Reply:
Generated cypress file, I thought it was needed, will be removing this

Reply:
Apparently, this is a mandatory file for cypress component testing